app/main.py

```diff
@@ -1,6 +1,20 @@
 from typing import Callable
+from functools import wraps
+from typing import Any, Tuple
```

Comment:
I think you can combine imports from typing in one line, like a "from typing import Callable, Any, Tuple" to make code more shorter

Reply:
Good point, fix it